src/Datadog.Trace/Metrics.cs

```diff
@@ -21,5 +21,11 @@ internal static class Metrics
         /// Read: Rate Limiter Priority Sample Rate
         /// </summary>
         public const string SamplingLimitDecision = "_dd.limit_psr";
+
+        /// <summary>
+        /// To be set on top-level spans
+        /// Top-level spans are spans with a different service name from their immediate parent
```

Comment:
```suggestion
        /// Identifies top-level spans.
        /// Top-level spans have a different service name from their immediate parent or have no parent.
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1303/commits/87ab80ddc9107b9b6cff3028a319b5824358d6e7